tests/tests/test_qtpy/test_interaction.py

```diff
@@ -1,39 +1,165 @@
+    def callback():
+        pass
+
+    # returns tuple of (combobox, combobox_label)
     combobox = add_combobox(
         layout,
         row=0,
         label=label,
-        items=["item 1", "item 2"],
+        items=items,
         label_stack=label_stack,
+        callback=callback,
     )
+
     assert combobox is not None
+    assert combobox[0].count() == len(items)
 
+    if label is None:
+        assert combobox[1] is None
+        assert layout.count() == 1
+    else:
+        assert combobox[1].text() == label
+        assert layout.count() == 2
 
-@pytest.mark.parametrize(
-    argnames="alignment", argvalues=["center", "left", "right"]
-)
-def test_add_button(alignment):
+
+@pytest.mark.parametrize("alignment", ["center", "left", "right"])
+def test_add_button(qtbot, box, alignment):
     """
     Smoke tests for add_button for all conditional branches
     """
-    layout = QGridLayout()
+    qtbot.addWidget(box)
+    layout = box.layout()
+    label = "A button"
+    tooltip = "A useful tooltip"
+
     button = add_button(
         layout=layout,
         connected_function=lambda: None,
-        label="A button",
+        label=label,
         row=0,
         alignment=alignment,
+        tooltip=tooltip,
     )
+
     assert button is not None
+    assert layout.count() == 1
+    assert button.text() == label
+    assert button.toolTip() == tooltip
+
+
+def test_add_checkbox(qtbot, box):
+    """
+    Smoke tests for add_checkbox for all conditional branches
+    """
+    qtbot.addWidget(box)
+    layout = box.layout()
+    label = "A checkbox"
+    tooltip = "A useful tooltip"
+
+    checkbox = add_checkbox(
+        layout=layout, default=True, label=label, tooltip=tooltip
+    )
+
+    assert checkbox is not None
+    # layout should contain 2 items: QLabel and QCheckbox
+    assert layout.count() == 2
+    assert layout.itemAt(0).widget().text() == label
+    assert checkbox.toolTip() == tooltip
+
+
+def test_add_float_box(qtbot, box):
+    """
+    Smoke tests for add_float_box for all conditional branches
+    """
+    qtbot.addWidget(box)
+    layout = box.layout()
+    label = "A float box"
+    tooltip = "A useful tooltip"
+    default = 0.5
+    minimum = 0.0
+    maximum = 1.0
+
+    floatbox = add_float_box(
+        layout=layout,
+        default=default,
+        minimum=minimum,
+        maximum=maximum,
+        label=label,
+        step=0.1,
+        tooltip=tooltip,
+    )
+
+    assert floatbox is not None
+    # layout should contain 2 items: QLabel and QDoubleSpinBox
+    assert layout.count() == 2
+    assert layout.itemAt(0).widget().text() == label
+    assert floatbox.maximum() == maximum
+    assert floatbox.minimum() == minimum
+    assert floatbox.value() == default
+    assert floatbox.toolTip() == tooltip
+
+
+def test_add_int_box(qtbot, box):
+    """
+    Smoke tests for add_float_box for all conditional branches
```

Comment:
```suggestion
    Smoke tests for add_int_box for all conditional branches
```
Could you clarify what "all conditional branches" means in each of the tests in this file by updating the docstrings?